pkg/pipeline/CdHandler.go

```diff
@@ -1016,13 +1026,18 @@ func (impl *CdHandlerImpl) converterWFR(wfr pipelineConfig.CdWorkflowRunner) pip
 	return workflow
 }
 
-func (impl *CdHandlerImpl) converterWFRList(wfrList []pipelineConfig.CdWorkflowRunner) []pipelineConfig.CdWorkflowWithArtifact {
+func (impl *CdHandlerImpl) converterWFRList(imageTaggingDataMap map[int]*ImageTaggingResponseDTO, wfrList []pipelineConfig.CdWorkflowRunner) []pipelineConfig.CdWorkflowWithArtifact {
 	var workflowList []pipelineConfig.CdWorkflowWithArtifact
 	var results []pipelineConfig.CdWorkflowWithArtifact
 	var ids []int32
 	for _, item := range wfrList {
 		ids = append(ids, item.TriggeredBy)
-		workflowList = append(workflowList, impl.converterWFR(item))
+		convertedWFR := impl.converterWFR(item)
+		if imageTaggingDataMap[convertedWFR.CiArtifactId] != nil {
+			convertedWFR.ImageReleaseTags = imageTaggingDataMap[convertedWFR.CiArtifactId].ImageReleaseTags
```

Comment:
``imageTaggingDataMap[convertedWFR.CiArtifactId]`` could have been extracted in if condition